sip-archetype/src/main/resources/archetype-resources/src/main/java/connectorgroups/__connectorGroup2__/connectors/AirQualityOutboundConnector.java

```diff
@@ -0,0 +1,73 @@
+package ${package}.connectorgroups.${connectorGroup2}.connectors;
+
+import ${package}.scenarios.definitions.GetAirQualityLatLonScenario;
+import ${package}.scenarios.models.AirQualityRequest;
+import ${package}.scenarios.models.AirQualityResponse;
+import de.ikor.sip.foundation.core.declarative.annonation.OutboundConnector;
+import de.ikor.sip.foundation.core.declarative.connector.GenericOutboundConnectorBase;
+import de.ikor.sip.foundation.core.declarative.model.UnmarshallerDefinition;
+import de.ikor.sip.foundation.core.declarative.orchestration.Orchestrator;
+import de.ikor.sip.foundation.core.declarative.orchestration.connector.ConnectorOrchestrationInfo;
+import de.ikor.sip.foundation.core.declarative.orchestration.connector.ConnectorOrchestrator;
+import java.util.Optional;
+import org.apache.camel.Exchange;
+import org.apache.camel.builder.EndpointProducerBuilder;
+import org.apache.camel.builder.endpoint.StaticEndpointBuilders;
+import org.apache.camel.component.jackson.JacksonDataFormat;
+import org.apache.camel.model.RouteDefinition;
+
+/**
+ * Outbound connector which calls
+ * GET https://air-quality-api.open-meteo.com/v1/air-quality?latitude={lat}&longitude={lon}&hourly=pm10,pm2_5
+ * The endpoint is used only for demo purposes
+ */
+@OutboundConnector(
+        connectorGroup = "Group2",
+        integrationScenario = GetAirQualityLatLonScenario.ID,
+        requestModel = String.class,
+        responseModel = AirQualityResponse.class,
+        connectorId = "AirQualityOutboundConnector")
+public class AirQualityOutboundConnector extends GenericOutboundConnectorBase {
+
+    // define external outbound endpoint
+    @Override
+    protected EndpointProducerBuilder defineOutgoingEndpoint() {
+        return StaticEndpointBuilders.http("https", "air-quality-api.open-meteo.com/v1/air-quality")
```

Comment:
The URI could be moved to the application.yml (if possible with archetype). Otherwise a comment would be great, that it should be configurable and how to achieve that.